lib/generators/decidim/templates/Gemfile.erb

```diff
@@ -14,6 +14,9 @@ ruby "<%= RUBY_VERSION %>"
 
 gem "decidim", <%= gem_modifier %>
 
+# Uncomment the following line if you want to use decidim-assemblies plugin
+# gem "decidim-assemblies", <%= gem_modifier %>
```

Comment:
Yeah, but features don't take an actual permanent section in the main menu (or the participatory space menu) unless they are actually used.

Perhaps we could check whether there's an actual assembly in order to show the main menu entry, but it feels really nice anyways to be able to plug & unplug plugins this easily.